src/Factories/JsonApiErrorFactory.php

```diff
@@ -18,16 +18,18 @@ class JsonApiErrorFactory implements JsonApiErrorFactoryInterface
     public static function createFromThrowable(Throwable $throwable): JsonApiError
     {
         $jsonError = new JsonApiError();
-        $jsonError->code = (string)$throwable->getCode();
+        $jsonError->code = (string) $throwable->getCode();
         $jsonError->title = $throwable->getMessage();
-        $jsonError->detail = $throwable->getTraceAsString();
+
+        // TODO NOW: Do we really always want to set this?
+        // $jsonError->detail = $throwable->getTraceAsString();
```

Comment:
hmm good one… the problem with not showing this all the time is to divide the "show" and "show not" cases I guess 🤔

Reply:
Maybe we should allow to set a `debug` parameter via config, that will add the details. 🤔